README.md

```diff
@@ -4,26 +4,6 @@
 
 This repository contains source code for the Java SDK for [Microsoft Azure Notification Hubs](https://azure.microsoft.com/en-us/services/notification-hubs/).
 
-## Note on FCM and GCM Support
-
-The Azure Notification Hubs SDK has support for Firebase Cloud Messaging (FCM) via the Legacy HTTP API which is compatible with GCM.  For registrations, use the `FirebaseRegistration` to create registrations, as the `GcmRegistration` class has been deprecated.  For more information, read the [Azure Notification Hubs and Google Firebase Cloud Messaging migration](https://aka.ms/AA9dpaz)
```

Comment:
Do we want to keep the link to the Google Firebase Cloud Messaging migration doc?

Reply:
Updated it! Thanks for the catch